pkg/validate/account_kubernetes.go

```diff
@@ -0,0 +1,63 @@
+package validate
+
+import (
+	"fmt"
+	"github.com/armory/spinnaker-operator/pkg/apis/spinnaker/v1alpha1"
+	"github.com/armory/spinnaker-operator/pkg/halconfig"
+	"github.com/armory/spinnaker-operator/pkg/validate/configfinder"
+)
+
+type kubernetesAccount struct {
+	Config interface{}
+}
+
+type kubernetesAccountValidator struct {
+	Account    *kubernetesAccount
+	SpinSvc    v1alpha1.SpinnakerServiceInterface
+	SpinConfig *halconfig.SpinnakerConfig
+	Options    Options
+}
+
+func (a *kubernetesAccount) GetName() string {
+	p := a.Config.(map[interface{}]interface{})
+	return p["name"].(string)
+}
+
+func (a *kubernetesAccount) GetChecksum() string {
+	p := a.Config.(map[interface{}]interface{})
+	return p["name"].(string)
```

Comment:
Checksum is supposed to be the checksum of all fields that make the config not just `name`. e.g. when you change `namespaces` etc.

Reply:
Yeah, I was thinking in checksum implementation to be done in the user story that actually implements the validation, rather than in this one that prepares the code to add validations. What do you think?